openmmapi/src/SplineFitter.cpp

```diff
@@ -37,6 +37,15 @@
 using namespace OpenMM;
 using namespace std;
 
+#define not_equal(a, b) (abs((a)-(b)) > 1e-15 + 1e-15*abs(b))  // same as scipy.interpolate()
```

Comment:
Preprocessor macros should be all upper case: `NOT_EQUAL()`.

Reply:
Or alternatively you could just make this a function rather than a macro:

```c++
static bool notEqual(double a, double b) {
  return (fabs(a-b) > 1e-15 + 1e-15*fabs(b));
}

Reply:
Done!